scripts/changes-include-dir

```diff
@@ -7,7 +7,7 @@
 # This compares the current branch to the master, so it might have some false
 # positives if using multiple layers of branches off of master.
 
-set -euo pipefail
+set -u
```

Comment:
I would be better to just put the command that tests for the diff in the `if` statement directly, which skips the `-e` check (as well as `pipefail` since that only affects the exit code of the pipeline).

Reply:
See the rules at https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin

Reply:
updated